samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -165,9 +165,20 @@ Location tracking
 All matters concerning location tracking are handled in the :file:`src/location_tracking.c` file.
 This involves setting up a periodic location request, and then passing the results to a callback configured by the :file:`src/application.c` file.
 
-.. note::
-  For location readings to be visible in the nRF Cloud portal, they must be marked as enabled in the `Device Shadow <nRF Cloud Device Shadows_>`_.
-  This is performed by the :file:`src/connection.c` file.
+For location readings to be visible in the nRF Cloud portal, they must be marked as enabled in the `Device Shadow <nRF Cloud Device Shadows_>`_.
+This is performed by the :file:`src/connection.c` file.
+
+Each enabled location method is tried in the following order until one succeeds: GNSS, Wi-Fi, then cellular.
+
+The GNSS and cellular location tracking methods are enabled by default and will work on both Thingy:91 and nRF9160 DK targets.
+
+.. _nrf_cloud_mqtt_multi_service_wifi_location_tracking:
+
+The Wi-Fi location tracking method is not enabled by default and requires an nRF9160 DK with an attached nRF7002 EK Wi-Fi evaluation kit.
```

Comment:
is `nRF7002 EK Wi-Fi evaluation kit` a bit redundant?

also... it can work on any hardware that has nRF7002 at its disposal and properly installed in the device tree

Reply:
> is nRF7002 EK Wi-Fi evaluation kit a bit redundant?

Not sure, technically the EK is part of the part number, IMO? I defer to @peknis on this

> also... it can work on any hardware that has nRF7002 at its disposal and properly installed in the device tree

Yeah, I need to word this better, LOCATION_METHOD_WIFI can work for any hardware that has nRF7002 at its disposal and properly installed in the device tree, but the convenience overlays only work specifically with the EK

Reply:
I've shifted the descriptions around so that they are actually, you know, _correct_.

Thanks for catching that! 

Also, I've truncated away the "Wi-Fi Evaluation Kit" from the description in its new location. @peknis is this still good with you?

Reply:
> is `nRF7002 EK Wi-Fi evaluation kit` a bit redundant?
> 
> also... it can work on any hardware that has nRF7002 at its disposal and properly installed in the device tree

Well, I think it can't work on all hardware with nRF7002 at present -- for example, the nRF7002-PDK is basically an nRF5340 + nRF7002 in a DK form-factor.  Since it doesn't have a 9160, MMSS will not yet work on it.

Reply:
Yep, EK Wi-Fi evaluation kit is a bit redundant. I'd say just EK should be enough.